content/en/apps/tutorials/local-setup.md

```diff
@@ -106,17 +128,37 @@ medic-conf --url=https://medic:password@localhost --accept-self-signed-certs
 
 Once you have run the above command it should complete with the message: `INFO All actions completed.`.
 
+Note: This will erase the default Maternal & Newborn Health Reference Application.
+
 <br clear="all">
 
  *****
 
-### 4. Upload Test Data
+### 5. Optional: Install Valid TLS Certificate  
 
-By default the CHT will have the [Maternal & Newborn Health Reference Application]({{< ref "apps/examples/anc" >}}) installed. To upload demo data you can use `medic-conf`:
+{{< figure src="local-ip.TLS.png" link="local-ip.TLS.png" class="right col-6 col-lg-8" >}}
 
-- Check out the [cht-core respository](https://github.com/medic/cht-core) to your local machine, either by using the [Github Desktop app](https://desktop.github.com/) or by running the following command in the directory you want to check the code out into: `git clone https://github.com/medic/cht-core.git`. This will create a `cht-core` directory.
-- Navigate your terminal to the `cht-core/config/default` directory. This is where the reference application is stored.
-- Run the following `medic-conf` command to compile and upload default test data to your local instance: `medic-conf --url=https://medic:password@localhost --accept-self-signed-certs csv-to-docs upload-docs`.
+With the blank project deployed to your CHT instance, you're ready to start writing your first app.  A big part of authoring an app is testing it on a mobile device, likely using the unbranded [Android APK](https://github.com/medic/cht-android).  In order to test in the APK, your CHT instance needs an valid TLS certificate which the default docker versoin does not have.
+
+To install a valid certificate, open a terminal in the `cht-core` directory. Ensure the `medic-os` container is running and make this call:
+
+```shell
+./scripts/add-local-ip-certs-to-docker.sh
+```
+
+You can see a `curl` call made before and after installing the certificate picture left.  The output of the script looks like this:
```

Comment:
Can you please rephrase this sentence?
"You can see a `curl` call made before and after installing the certificate **picture left.**"

Reply:
For sure!  See d5c7f44